oam/install_scripts/post-install

```diff
@@ -190,22 +216,16 @@ if [ $user = "root" ]; then
     touch $RCFILE    
     chmod +x $RCFILE
 else
-    $SUDO touch $RCFILE
-    $SUDO chmod 777 $RCFILE
-    $SUDO printf '%s\n' '#!/bin/bash' "#" | $SUDO tee -a $RCFILE > /dev/null 2>&1
-    
-    if [ -n "$systemctl" ]; then
-	$SUDO systemctl start rc-local >/dev/null 2>&1
-	$SUDO systemctl enable rc-local >/dev/null 2>&1
-    fi
+    printf '%s\n' '#!/bin/bash' "#" | tee -a $RCFILE > /dev/null 2>&1
```

Comment:
note to self.  working here.  I can't tell what we're trying to do here.  This will fail, but that's good, because we seem to be trying to add #!/bin/bash to the end of the rc.local init script..?